src/main/java/dev/goral/rpgmanager/rpgSystems/cpRed/characters/weapons/CpRedWeaponsDTO.java

```diff
@@ -0,0 +1,22 @@
+package dev.goral.rpgmanager.rpgSystems.cpRed.characters.weapons;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+@AllArgsConstructor
+@Getter
+@ToString
+public class CpRedWeaponsDTO {
+    private String requireSkillId;
```

Comment:
```suggestion
    private String requiredSkillId;
```

sugeruje ponownie jak w klasie bazowej